Library/ViewModels/CommentComposerViewModel.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public typealias CommentComposerViewData = (avatarURL: URL?, isBackingProject: Bool)
+
+public enum CommentComposerConstant {
+  public static let characterLimit: Int = 9_000
```

Comment:
This is fine but should probably be accompanied by an inline comment about where the number is coming from.

Reply:
How about `// From feature requirements`?

Reply:
Connected on Slack. Going to resolve this.